contracts/interfaces/IStableDebtToken.sol

```diff
@@ -14,17 +14,17 @@ interface IStableDebtToken is IInitializableDebtToken {
    * @dev Emitted when new stable debt is minted
    * @param user The address of the user who triggered the minting
    * @param onBehalfOf The recipient of stable debt tokens
-   * @param amount The amount minted (user entered amount + balance increase from interest)
-   * @param currentBalance The current balance of the user
-   * @param balanceIncrease The increase in balance since the last action of the user
+   * @param value The amount minted (user entered amount + balance increase from interest)
+   * @param currentBalance The balance of the user based on the previous balance and balance increase from interest
```

Comment:
The name `currentBalance` is totally misleading imho, since its not clear what "current" refers and its not providing either the previous balance or the new one (its providing a kind of intermediary balance: previousBalance+interest).

I would be in favor of renaming to a clearer term.

Reply:
It refers to the current balance of the user without taking the performed action into account